api-tester/src/main/java/com/revenuecat/apitester/kotlin/PurchasesAPI.kt

```diff
@@ -198,6 +199,8 @@ private class PurchasesAPI {
         val customerInfo: CustomerInfo = purchases.awaitCustomerInfo()
         val customerInfoFetchPolicy: CustomerInfo =
             purchases.awaitCustomerInfo(fetchPolicy = CacheFetchPolicy.FETCH_CURRENT)
+
+        val offerings: Offerings = purchases.awaitOfferings()
```

Comment:
It’s weird that this doesn’t need a `catch`. Is it a `RuntimeException`?

Reply:
It's not `RuntimeException`. Kotlin doesn't have checked exceptions, that's why there's no need to catch here. Take a look at this https://kotlinlang.org/docs/exceptions.html#checked-exceptions

Reply:
Oh yikes. Okay 🫤